src/app/modules/core/advertisement/components/latest/latest-container/latest-container.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core'
+import { Observable } from 'rxjs'
+
+import { AdvertisementService } from './../../../advertisement.service'
+
+@Component({
+  selector: 'cav-latest-container',
+  templateUrl: './latest-container.component.html',
+  styleUrls: ['./latest-container.component.scss'],
+})
+export class LatestContainerComponent implements OnInit {
+  constructor(private advService: AdvertisementService) {}
+  start = 0
+  end = 2
+
+  latest: Observable<any[]>
```

Comment:
Anche qui il tipo non è sempre `IMockAdvertisement[]` ?
e cambiare il nome in latestAdvertisement? è più specifico così secondo me